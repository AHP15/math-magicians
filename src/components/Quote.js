import NavBar from './NavBar';

const Quote = () => (
  <div style={{ height: '100vh', display: 'grid', placeItems: 'center' }}>
    <NavBar />
    <h1 style={{ width: '100%', maxWidth: '700px', textAlign: 'center' }}>
      Mathematics is not about number, equations, computations,
      or algorithms: it is about understanding. -William Paul Thruston
    </h1>
  </div>
);
export default Quote;
