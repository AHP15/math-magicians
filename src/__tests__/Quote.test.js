import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../components/Quote';

it('renders Quote correctly', () => {
  const tree = renderer
    .create(<BrowserRouter><Quote /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
