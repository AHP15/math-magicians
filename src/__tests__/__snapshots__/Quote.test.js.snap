// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Quote correctly 1`] = `
<div
  style={
    Object {
      "display": "grid",
      "height": "100vh",
      "placeItems": "center",
    }
  }
>
  <nav
    className="navbar"
  >
    <a
      className="link"
      href="/"
      onClick={[Function]}
    >
      <h1>
        Math Magicians
      </h1>
    </a>
    <ul
      className="links"
    >
      <li>
        <a
          className="link"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
  <h1
    style={
      Object {
        "maxWidth": "700px",
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    Mathematics is not about number, equations, computations, or algorithms: it is about understanding. -William Paul Thruston
  </h1>
</div>
`;
